pkg/up/up.go

```diff
@@ -715,20 +731,25 @@ func (u *upRunner) run() error {
 		if err != nil {
 			return err
 		}
-		allPodsReady := true
-		for app := range u.appsThatNeedToBeReady {
-			if app.maxObservedPodStatus != podStatusReady {
-				allPodsReady = false
-			}
-		}
-		if allPodsReady {
+
+		if u.checkIfPodsReady() {
 			break
 		}
 	}
 	fmt.Printf("pods ready (%d/%d)\n", len(u.appsThatNeedToBeReady), len(u.appsThatNeedToBeReady))
 	return nil
 }
 
+func (u *upRunner) checkIfPodsReady() bool{
+	allPodsReady := true
+	for app := range u.appsThatNeedToBeReady {
+		if !(app.maxObservedPodStatus == podStatusReady || app.maxObservedPodStatus == podStatusCompleted)  {
```

Comment:
Can simplify condition to `app. maxObservedPodStatus < podStatusReady `

Reply:
Fixed